jest.config.js

```diff
@@ -61,9 +61,7 @@ module.exports = {
   "moduleFileExtensions": ["js", "jsx", "json", "ts", "tsx"],
 
   // A map from regular expressions to module names that allow to stub out resources with a single module
-  moduleNameMapper: {
-    "^src/(.*)$": "<rootDir>/src/$1",
-  },
+  // moduleNameMapper: {},
```

Comment:
why not just remove this line?

Reply:
Config file magic?? I don't know ;-)

@justinbarry ?